.gitignore

```diff
@@ -5,6 +5,7 @@ docs
 vite-bundle-visualizer.html
 .npm-packages
 .pnp.*
+.vscode/*
```

Comment:
The extensions.json is still part of the repo. This entry only prevents new files being checked in. I had some changes in vscode and it added some files there that should not go into the repo. We can add an explicit exception for extensions.json but since it is already in the repo this should not be necessary.